packages/container/src/dom/createContainerMessageListener.ts

```diff
@@ -9,13 +9,14 @@ import {
   SetModalOpen,
   SetContent,
   isGetContextMessage,
+  PluginLoadedMessage,
 } from '@storyblok/field-plugin'
 
 type ContainerActions = {
   setHeight: (height: number) => void
   setContent: SetContent
   setModalOpen: SetModalOpen
-  setPluginReady: () => void
+  setPluginReady: (message: PluginLoadedMessage) => void
```

Comment:
We could consider adhering to this pattern in the other event handlers (instead of using `SetContent`, `SetModalOpen` etc), but I leave it for another pull request.